frontend/index.html

```diff
@@ -468,8 +529,91 @@ <h2>Contact</h2>
         const study_name = first_domain.substr(0,openpath_index);
         return study_name;
       }
+
+      var curr_lang = "en"; // Default language is set to English
+
+      function getLang(){
+
+      var selectBox = document.getElementById("buttonBox");
+      var selectedValue = selectBox.options[selectBox.selectedIndex].value;
+      curr_lang = selectedValue;
+      localStorage.setItem("langValue", curr_lang);
+
 			$(function() {
 				$(document).ready(function() {
+
+					const STUDY_NAME = _getStudyName(window.location);
+          const JOIN_CODE = `nrelopenpath://join_study?label=${STUDY_NAME}&source=github`
+
+          $.getJSON(`https://raw.githubusercontent.com/e-mission/nrel-openpath-deploy-configs/main/configs/${STUDY_NAME}.nrel-op.json`, function(data) {
```

Comment:
Wait, you have two instances of the call? Why? The original call already gets both languages. Why do you need to make a second (or if the user switches back and forth multiple times, `n` different calls to read The Same Data)?

Reply:
Yes. We are still experimenting with this section. Whenever the user changes language, the button will activate a function in which it will call the instance ( in this case, the second one that we created that is inside the function `getLang()` ) making all the variables reload/ refresh and obtain the translated text from the config files without having a page reload. The code that is inside that function is pretty much a copy/paste of the all the values previously obtained. Now, the reason we left the original instance at the end of the code is because that instance will be called whenever the page makes its first load ( whenever the user opens the page for the first time ). 

We found a more efficient way to counter this, but we are still working on it.

Reply:
If the user switches back and forth multiple times then the function will keep activating and calling the data again and again. The function will retrieve the desired translated text.

Reply:
Also, originally, we where planning on wrapping the default load code in a function and then calling the function at the end of the html. So that when the user opens the page for the first time it will instantly call the function to get the text in English from the config files (English is set as the default language, this can be change by changing the value of `curr_lang ` ). We decided to generate a separate function so that we can do some experimenting.